bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -97,6 +100,15 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function
             cont.append(tempSelect);
             me._selections.push(select);
         });
+        // map to numbers so we can filter
+        var numMap = me.regionsetRestrictions.map(function (iter) {
```

Comment:
Move this string -> number mapping to indicatorSelected() where me.regionsetRestrictions is set. Then you don't need the numMap variable and can use me.regionsetRestrictions instead.